graphein/ml/datasets/torch_geometric_dataset.py

```diff
@@ -237,6 +264,7 @@ def download(self):
             ]
 
     def __len__(self) -> int:
+        """Returns length of data set (number of structures)."""
```

Comment:
We should return the number of examples (not just the number of structures for the multiple chain reason I mentioned previously)

Reply:
Ok